README.md

```diff
@@ -1,2 +1,55 @@
-# teamvolik
-Telegram bot for signing up for volleyball games
+[![Hits-of-Code](https://hitsofcode.com/github/teamvolik/teamvolik?branch=main)](https://hitsofcode.com/github/teamvolik/teamvolik/view)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/teamvolik/teamvolik.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/teamvolik/teamvolik/context:python)
+![coverage](https://img.shields.io/codecov/c/github/teamvolik/teamvolik?color=080)
+![lines](https://img.shields.io/tokei/lines/github/teamvolik/teamvolik?color=dgreen)
+![pr_closed](https://img.shields.io/github/issues-pr-closed/teamvolik/teamvolik?color=FFF)
+![last_commit](https://img.shields.io/github/last-commit/teamvolik/teamvolik)
+![release](https://img.shields.io/github/v/release/teamvolik/teamvolik)
+![license](https://img.shields.io/github/license/teamvolik/teamvolik?color=0077FF)
+
+# teamvolik telegram bot
+<img src=".github/README_MISC/teamvolik.jpeg" alt="teamvolik_logo" width="200">
+
+
+Telegram bot, which greatly simplifies registration for local beach volleyball games.
+
+## Formulation of the problem
```

Comment:
Shouldn't it be something like
`What's inside' or 'What is planned'?
I don't like `formulation` word here